samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/services/GradingService.java

```diff
@@ -2307,32 +2307,26 @@ public double getTotalCorrectScore(ItemGradingData data, Map publishedAnswerHash
   }
 
   private void setIsLate(AssessmentGradingData data, PublishedAssessmentIfc pub){
-	  // If submit from timeout popup, we don't record LATE
-	  if (data.getSubmitFromTimeoutPopup()) {
-		  data.setIsLate(false);
-	  }
-	  else {
-		  Boolean isLate = false;
-		  AssessmentAccessControlIfc a = pub.getAssessmentAccessControl();
-		  if (a.getDueDate() != null && a.getDueDate().before(new Date())) {
-			isLate = Boolean.TRUE;
-		  } else {
-			isLate = Boolean.FALSE;
-		  }
+    Boolean isLate;
```

Comment:
personally i would initialize to FALSE and then just remove lines 2139-2321 .... but doesn't really matter

Reply:
Agreed; done.